### PR TITLE
feat: add Spanish helper method

### LINKED ISSUE #32: Add a helper method for Spanish Language
See #1 on how to implement it. It's very simple and it's the perfect candidate for the first issue collaboration.

Feel free to post any questions.

Code example: https://github.com/chgasparoto/aws-polly-voices/pull/2/files

### PR DESCRIPTION
**Pull request by masihtak**
## Proposed changes
Added Spanish helper method
fix #32

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/chgasparoto/aws-polly-voices/blob/master/CONTRIBUTING.md) doc
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -82,6 +82,10 @@ export default class Voices {
     return this.byLangCode('pl-PL')
   }
 
+  spanish () {
+    return this.byLangCode('es-ES')
+  }
+
```
Comment (chgasparoto) [inline src/index.js]: For Spanish language in general you should use the method `byLang`. Take a look at `portuguese` or `english` method.

In this way you are getting only `Castilian Spanish`

File: test/index.test.js [hunk 2]
```diff
@@ -328,6 +335,19 @@ describe('Voices', () => {
 
     expect(spy).toHaveBeenCalledWith('pl-PL')
 
+    spy.mockRestore()
+  })
+  test('it should return SPANISH voice data', () => {
+    const spy = jest.spyOn(voices, 'byLangCode')
+    const spanish = voices.polish()
```
Comment (chgasparoto) [inline test/index.test.js]: should be `voices.spanish`

File: test/index.test.js [hunk 3]
```diff
@@ -328,6 +335,19 @@ describe('Voices', () => {
 
     expect(spy).toHaveBeenCalledWith('pl-PL')
 
+    spy.mockRestore()
+  })
+  test('it should return SPANISH voice data', () => {
+    const spy = jest.spyOn(voices, 'byLangCode')
```
Comment (chgasparoto) [inline test/index.test.js]: should be `byLang`

File: test/index.test.js [hunk 4]
```diff
@@ -328,6 +335,19 @@ describe('Voices', () => {
...
+    const spanish = voices.polish()
+
+    expect(spanish.val).toBeArray()
+    expect(spanish.val[0]).toBeObject()
+    expect(spanish.val[0]).toContainAllKeys(voiceObjectKeys)
+    expect(spanish.id).toBeString()
+
+    expect(spy).toHaveBeenCalledWith('es-ES')
```
Comment (chgasparoto) [inline test/index.test.js]: should be something like `Spanish`

**Comment by masihtak** [comment]
Hi @chgasparoto  sorry for that, I fixed the issue.

File: src/index.js [hunk 5]
```diff
@@ -82,6 +82,10 @@ export default class Voices {
     return this.byLangCode('pl-PL')
   }
 
+  spanish () {
+    return this.byLang('Castilian Spanish')
```
Comment (chgasparoto) [inline src/index.js]: should be only `Spanish` here, not `Castilian Spanish`. Because it's Spanish in general and must return all Spanish variants.

Comment (chgasparoto) [reply src/index.js]: That's why the test is failing

**Comment by chgasparoto** [comment]
@masihtak please update the README.md file with your implementation.

**Comment by chgasparoto** [comment]
Code looks good, you just need to fix the conflict with README.md file, please.

**Comment by masihtak** [comment]
What changes needed for the README.md file ? I already added implementation

**Comment by chgasparoto** [comment]
@masihtak you are right, there's no implementation to do but there are conflicts to be resolved before the merge. See below 👇

**Comment by masihtak** [comment]
It's because branch, isn't?

**Comment by chgasparoto** [comment]
yes, I've merged other branches into master, so it's normal to get some conflicts.

**Comment by chgasparoto** [comment]
I'll wait one more week for the conflict fix otherwise I'm going to close it 😉

**Pull request merged**